### DISCUSSION TITLE
Use in Node.js

### DISCUSSION THREAD

**Discussion by narimanlotfi** [majidh1/JalaliDatePicker#40] [Question] [answered]
نحوه استفاده از دیت پیکر در یک سرور Node.js چطور هست؟ من نمیخوام بعد از لود هر State یکبار Watch رو بزنم.

**Comment by majidh1** [comment, marked as answer]
تفاوتی نداره از چه سروری استفاده کنید. نیاز نیست بعد از ساخت هر اینپوت Watch کنید و فقط یکبار در لود App کافی میباشد. 
نکته: data-jdp روی input فراموش نشه
